lighthouse-core/lib/i18n/i18n.js

```diff
@@ -358,143 +344,140 @@ function getRendererFormattedStrings(locale) {
 }
 
 /**
- * Register a file's UIStrings with i18n, return function to
- * generate the string ids.
- *
+ * Returns a function that generates `LH.IcuMessage` objects to localize the
+ * messages in `fileStrings` and the shared `i18n.UIStrings`.
  * @param {string} filename
  * @param {Record<string, string>} fileStrings
  */
 function createMessageInstanceIdFn(filename, fileStrings) {
```

Comment:
can we rename this to `createIcuMessageFn` internally here.. we can still expose it in module.exports as the long name (for now)

Reply:
> can we rename this to `createIcuMessageFn` internally here.. we can still expose it in module.exports as the long name (for now)

should both names be exposed and we convert all Lighthouse audits over to using `createIcuMessageFn`?